tools/cldr-code/src/test/java/org/unicode/cldr/util/TestVettingViewer.java

```diff
@@ -52,19 +51,19 @@ protected final PathHeader getPathHeader(String xpath) {
         }, "hello world");
 
         Organization usersOrg = Organization.surveytool;
-        final boolean quick = false;
-        final Factory baselineFactory = CLDRConfig.getInstance().getCldrFactory();
-        final Factory sourceFactory = baselineFactory;
-        vv.setBaselineFactory(baselineFactory);
+
+        final Factory sourceFactory = CLDRConfig.getInstance().getCldrFactory();
+
+        Factory baselineFactory = sourceFactory;
 
         EnumSet<VettingViewer.Choice> choiceSet = EnumSet.of(VettingViewer.Choice.englishChanged);
         CLDRFile sourceFile = sourceFactory.make(loc, true);
         CLDRFile baselineFile = baselineFactory.make(loc, true);
-        Relation<R2<SectionId, PageId>, VettingViewer<Organization>.WritingInfo> file;
+        Relation<R2<SectionId, PageId>, VettingViewer<Organization>.WritingInfo> sorted;
```

Comment:
This was named "sorted" in some places and "file" in other places. Changed to use the name "sorted" consistently. Neither name actually means anything to me. The type `Relation<R2<SectionId, PageId>, VettingViewer<Organization>.WritingInfo>` occurs repeatedly. I would need to spend a lot more time to have any hope of understanding it. Such complexity cries out for encapsulation!

Reply:
I can never remember how it works until i step through it :joy: